src/SolidStateDetector/Contacts.jl

```diff
@@ -46,8 +46,21 @@ struct Contact{T,G,MT} <: AbstractContact{T}
     id::Int
     name::String
     geometry::G
+    symmetry::Dict
```

Comment:
No. This will make the `struct Contact` type unstable. We don't need a `Dict` here.

We have to create structs for the symmetries. 

While I am writing this, I just realized that this actually does not belong to the contact itself. 
As the symmetries might depend on the environment of the detector. 
We need to add a vector of Symmetries (or a `Dict`) to the `Simulation` I think.